src/twisted/python/test/test_deprecate.py

```diff
@@ -961,7 +962,12 @@ def checkPassed(self, func, *args, **kw):
         @return: L{_passed}'s return value
         @rtype: L{dict}
         """
-        return _passed(inspect.getargspec(func), args, kw)
+        if getattr(inspect, "signature", None):
```

Comment:
(required)
The docstring above refers to `passed` which no longer exists (and .. didn't before, since it was `_passed`).

(optional)
I kinda also want to say something about the duplication between the logic here and the logic inside `_mutuallyExclusiveArguments`.  It seems like it might be better if there were just one implementation of that idea and these tests could exercise it.